src/test/java/org/jpeek/web/DyNumTest.java

```diff
@@ -90,18 +90,16 @@ public void testConstructDyNumFromDouble() {
         final DyNum dynum = new DyNum(number);
         new Assertion<>(
             "Double values should be equal.",
-            () -> dynum.doubleValue(),
+            dynum::doubleValue,
```

Comment:
get rid of intermediate dynum and number variables here
just `new DyNum(2.0f)::floatValue` etc

Reply:
I didn't pass 2.0f and stayed with `number` because of later comparison with this value and static checks for duplicates of literals and numbers.

Reply:
Ok, that's fine